docs/api/field_type_reference.md

```diff
@@ -1501,6 +1501,157 @@ new FieldType\Value([
 ]);
 ```
 
+### GraphQL Field Type operations
+
+To get a Matrix Field Type, you will need to specify a Content ID, a Content Type, and a Field Type. 
```

Comment:
Is the fact that you need a Content ID relevant here ? First, it is not accurate, as you can ask for all items of a type without an ID. Second, the fact that you need a content id to get an item isn't specific to Matrix.